### PR TITLE
Make error messages to be consistent for all commands

### PR DESCRIPTION
**Pull request by suganyasuven**
Fixes https://github.com/ballerina-platform/ballerina-lang/issues/20821

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/InitCommand.java [hunk 1]
```diff
@@ -101,19 +106,21 @@ public void execute() {
...
                     null,
                     false);
+            CommandUtil.exitError(this.exitWhenFinish);
             return;
         }
 
         // Check if the template exists
         if (!template.equals("") && !CommandUtil.getTemplates().contains(template)) {
             CommandUtil.printError(errStream,
-                    "Template not found, use `bal init --help` to view available templates.",
```
Comment (azinneera) [inline cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/InitCommand.java:114]: Should we have periods or not? @praneesha

File: cli/ballerina-cli/src/main/java/io/ballerina/cli/launcher/Main.java [hunk 2]
```diff
@@ -462,14 +472,23 @@ private String promptForInput(String msg) {
         @CommandLine.Option(names = { "--version", "-v" }, hidden = true)
         private boolean versionFlag;
 
+        @CommandLine.Parameters
+        private List<String> argList = new ArrayList<>();
```
Comment (azinneera) [inline cli/ballerina-cli/src/main/java/io/ballerina/cli/launcher/Main.java:476]: Why so we introduce this arglist?

File: cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/InitCommand.java (hunk 1 above)
Comment (praneesha) [reply cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/InitCommand.java:114]: Better to use the period for complete sentences.

File: cli/ballerina-cli/src/main/java/io/ballerina/cli/launcher/Main.java (hunk 2 above)
Comment (suganyasuven) [reply cli/ballerina-cli/src/main/java/io/ballerina/cli/launcher/Main.java:476]: Mistakenly added this change here. Will revert this.

**Review by praneesha — APPROVED** [review]

**Review by azinneera — APPROVED** [review]

**Pull request merged**